content/articles/flow-and-typescript-2.md

```diff
@@ -0,0 +1,108 @@
+Title: Flow and TypeScript part 2: TypeScript
+Date: 2015-11-20
+Short_summary: Static typing helps keeping codebases (and developers) sane. We have a look on adding types to JavaScript
+Category: Dev
+Authors: Vincent, Tom
+
+
+Continuing from the [first part that introduced Flow](https://blog.wearewizards.io/flow-and-typescript-part-1-flow), we are now going to have a look at [TypeScript](http://www.typescriptlang.org/).
+
+TypeScript is a superset of JavaScript that compiles to several targets, including ES5 and ES6. The current latest version of TypeScript (1.6 at the time of writing) now supports JSX, which was the main blocker for most people using React.
+
+One of the great advantages of TypeScript is  [DefinitelyTyped](https://github.com/borisyankov/DefinitelyTyped). This repo contains definition for over a thousand JavaScript packages but more on that later.
+
+Here is the [repo](https://github.com/Keats/flow-typescript/tree/master/typescript). Note that I have used TypeScript for a project before.
```

Comment:
We've implemented the same project once in Flow and once in TS. Find the repo [here](...) 

(readers won't have context what the repo means and need that info. even if they read the flow post they won't remember)